CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format of this Changelog is based on [Keep a Changelog](https://keepachangel
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased-`x.y.z`] - 2019-xx-xx
+- Adding CN endpoints to DeploymentLauncher
```

Comment:
```suggestion
- You can now launch cloud deployments in China by specifying the `CN` region in the Deployment Launcher.
```

Reply:
This format better conveys WIIFM as per: https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md#guiding-principles